tests/integration/multipass/test_launch.py

```diff
@@ -57,21 +57,14 @@ def core20_instance(instance_name):
             instance.delete()
 
 
-@pytest.mark.parametrize(
```

Comment:
Similar to what I did for LXD, I moved this parametrization to the `test_multipass_provider.py` module